Test/client_test.py

```diff
@@ -34,22 +34,23 @@ def test_get_org(self):
         response = self.c.get_org("a_myOrg84")
         self.assertEqual("a_myOrg84",response['org_name'])
 
-    def test_edit_org(self):
-        orgname="a_myOrg84"
-
-        obj = {
-        "id": "b85d8c4e-7f3c-40b9-9659-6adc2cb0e16f",
-        "org_name": "a_myOrg84",
-        "org_label": "some label",
-        "org_contact_name": "Some owner",
-        "org_email": "a_myOrg84@algo.com",
-        "org_created_at": "2020-11-30T23:51:40",
-        "type_id": "3d40e3b0-d82a-11ea-9a3c-0ee5e2d35097",
-        "resource_type": "organization"
-        }
-
-        response = self.c.edit_org(orgname,obj)
-        self.assertEqual(204,response.status_code)
+
```

Comment:
@kennydaniel @ljessen1 @gwallace-algorithmia you three approved this PR. What context am I missing about it being ok to completely comment out a regression test? There isn't any discussion of this in the PR description.

Reply:
@dinoboy197 An unusual request was made of me to do a PR in this project which I have never looked at before. I assumed it was something time sensitive and that I was being asked to do it because I was the closest warm body. I also assumed the people involved knew what they were doing. Did I fail a test? :(

Reply:
Thanks for the info Gordon!

From your description, we have a couple of improvements to make procedurally for the future. My comments below are intended for Algorithmia engineering as a whole, not specifically for you :) This is a learning moment :bulb: !

On the test removal -
* there is no description in the PR, or comment on the lines of code, about why a regression test is being commented out. Even a one-sentence description would help to justify this for the record, so that everyone can see and have an audit record of what's going on.
* I don't think we have this stated formally anywhere, but I'll assert that removing regression tests need to be justified and reviewers must agree with the justification. I realize this work was a "rush order" to assist with a customer incident, but by removing a regression test, we risk introducing _other_ bugs both into this version and all other versions. That cannot be acceptable - we can't trade one bug for the possibility of another :)

On the "find a warm body to press a button" -
* PRs need to be reviewed by someone with context. Finding a "close by engineer" to simply press the approve button isn't acceptable, especially because assuming people involved know what they're doing is a faulty assumption. I mean no disrespect to John for putting in this patch (thanks John!) Rather, the entire reason that we have a peer review process for code changes is because we need an outside reviewer to give a critical eye to changes to ensure that nothing suspicious is going on (like removing regression tests!)